seabird.proto

```diff
@@ -27,6 +27,29 @@ message Event {
     PerformActionEvent perform_action = 9;
     PerformPrivateActionEvent perform_private_action = 10;
   }
+
+  // Tags are a set of key-value items which may expose additional information
+  // about the given Event.
+  //
+  // All tags SHOULD be prefixed with something unique to that plugin or
+  // backend. All tags starting with `seabird/` are reserved for use by
+  // seabird-core.
+  //
+  // Currently known tags:
+  // - proxy/skip - this can be set to '1' if the message should not be
+  //   proxied. Any other value means the message should be proxied. As an
+  //   example, this is useful for metadata which some backends may already
+  //   provide. Generally if a plugin uses this, it should respond to both
+  //   normal events and the "send" variant of that message.
+  // - proxy/source-channel-id - set by the proxy plugin if a message is
+  //   proxied, it will contain the original source channel ID
+  // - proxy/source-channel-name - set by the proxy plugin if a message is
+  //   proxied, it will contain the original source channel name
+  // - proxy/source-user-id - set by the proxy plugin if a message is proxied,
+  //   it will contain the original source user ID
+  // - proxy/source-user-name - set by the proxy plugin if a message is proxied,
+  //   it will contain the original source user name.
```

Comment:
Maybe a hierarchy?

```
proxy/
  source/
    channel/
      id
      name
    user/
      id
      name
```

So, tags would be `proxy/source/channel/name`. Maybe easier to parse?

Reply:
I'm marking these as "proposed" so I can decide later and not lose the documentation